enterprise/server/util/vfs_server/vfs_server.go

```diff
@@ -992,8 +983,11 @@ func (p *Server) SetAttr(ctx context.Context, request *vfspb.SetAttrRequest) (*v
 
 	node.mu.Lock()
 	defer node.mu.Unlock()
+
+	newAttrs := proto.Clone(node.attrs).(*vfspb.Attrs)
```

Comment:
could be nice to add a comment about why updateAttrs is not used here -- i assume it's because there's too much extraneous logic you don't want in the closure?